src/main/java/com/fasterxml/jackson/databind/deser/std/CollectionDeserializer.java

```diff
@@ -16,6 +11,9 @@
 import com.fasterxml.jackson.databind.jsontype.TypeDeserializer;
 import com.fasterxml.jackson.databind.type.LogicalType;
 import com.fasterxml.jackson.databind.util.ClassUtil;
+import java.io.IOException;
+import java.util.*;
+import java.util.Objects;
```

Comment:
Import order changes are unfortunate, they're the result of applying automated fixes as documented in the commit message: 58b407d72456f66c03e4e704046704bf3c9fbcc0